Dockerfile

```diff
@@ -1,7 +1,9 @@
-FROM mhart/alpine-node:latest 
-RUN npm install -g s3rver@2.2.8
+FROM mhart/alpine-node:latest
+RUN apk add --no-cache git
+RUN npm install -g allanlegalstart/s3rver#pour_toi_public
```

Comment:
I'm not sure I wanna see private jokes on my favorite projects

Reply:
All right my bad. There really is some "pour_toi_public" branch on the repo of this lunatic :O. I guess it's not meant to be merge this way, right ?